### ISSUE TITLE
Disabled languages still accessible

### ISSUE THREAD

**Issue by alexiswbr** [Laravel-Backpack/langfilemanager#32]
When you set the `active` value to false for a language, this language still appear when you edit the translations files.
I think it is because of this line :

```php
// Backpack\LangFileManager\app\Http\Controllers\LanguageCrudController
$this->data['languages'] = $languages->orderBy('name')->get();

// fixed with

this->data['languages'] = $languages->orderBy('name')->where('active', 1)->get();
```

I made a pull request, don't really know how to assign it to this issue

**Comment by lloy0076** [comment]
https://laravel-backpack.readme.io/docs/crud-fields#section-range

**Comment by lloy0076** [comment]
Closing - was merged in #33.
